src/metatensor/models/cli/eval.py

```diff
@@ -201,38 +227,41 @@ def eval_model(
             dtype=dtype,
         )
 
-        # Predict targets
         if hasattr(options, "targets"):
+            # in this case, we only evaluate the targets specified in the options
             eval_targets = read_targets(options["targets"], dtype=dtype)
-            eval_dataset = Dataset(system=eval_systems, energy=eval_targets["energy"])
-            _eval_targets(model, eval_dataset)
+            eval_outputs = {
+                target: tensormaps[0].block().gradients_list()
+                for target, tensormaps in eval_targets.items()
```

Comment:
Not sure I understand this line, but why do we call `gradients_list()` here instead of just using `block().values`? Is it because the forces and stresses are stored as gradients of energies? What if we need energy only?

Reply:
We're feeding the function a list of gradients to compute for each target (this information is sadly not contained in the evaluation options)